test/fuzz/random.h

```diff
@@ -0,0 +1,79 @@
+#include <random>
+#include <vector>
+
+namespace Envoy {
+namespace Random {
+
+class PsuedoRandomGenerator64 : public RandomGenerator {
+public:
+  PsuedoRandomGenerator64() = default;
+  ~PsuedoRandomGenerator64() override = default;
+
+  void initializeSeed(uint64_t seed) { prng_ = std::make_unique<std::mt19937_64>(seed); }
```

Comment:
Added ASSERT. Yeah, I think it's ideal as well to get it from constructor, but for my use case this is better option. I can add a constructor as well, but I think you mentioned only add what you need for now.

Reply:
Just throwing this out there because I was rethinking this, do you think it's better for the load balancer base to take a seed in constructor? 
That way it's exposed that it contains some seeded randomness required for LB fuzzing

Reply:
actually disregard, i think both are fine since you have an explicit initializeLbComponents!